torchvision/prototype/datasets/_builtin/stanford-cars.categories

```diff
@@ -0,0 +1,196 @@
+AM General Hummer SUV 2000
```

Comment:
How did you generate this file? Unless you need to compile it manually, the best way is to put a `_generate_categories` method on the dataset and use the `torchvision/prototype/datasets/generate_categories_file.py` script. This way we have a record of the code that was used to generate the file.

Reply:
I did not add the `generate_categories` method in the dataset because we will have to download `car_devkit.tgz` irrespective of the split(train/test.) 
That's why I did not include it. 
But if you insist, I will :)

Reply:
The method is not executed at runtime. So you only need this file locally if you want to generate the file. And this is a compromise I'm willing to make to have a reproducible generation. Make sure you use something like

https://github.com/pytorch/vision/blob/50608fbc263da386ad7abf7b32bd32ed7f691170/torchvision/prototype/datasets/_builtin/imagenet.py#L193-L195

to actually get the resources that include the file you need.